crates/topos-tce-storage/src/validator/mod.rs

```diff
@@ -213,11 +243,19 @@ impl ValidatorStore {
                 .pending_pool_index
                 .insert(&certificate.id, &id)?;
 
+            debug!(
+                "Certificate {} is now in the pending pool at index: {}",
+                certificate.id, id
+            );
             Ok(Some(id))
         } else {
             self.pending_tables
                 .precedence_pool
                 .insert(&certificate.prev_id, certificate)?;
+            debug!(
+                "Certificate {} is now in the precedence pool, {} isn't delivered yet",
```

Comment:
```suggestion
                "Certificate {} is now in the precedence pool, because the previous cert with id {} isn't delivered yet",
```

maybe?